src/execute_request.jl

```diff
@@ -39,13 +39,13 @@ MIME types that when rendered (via stringmime) return JSON data. See
 This is necessary to embed the JSON as is in the displaydata bundle (rather than
 as stringify'd JSON).
 """
-const ijulia_jsonmime_types = Vector{Union{MIME, Vector{MIME}}}([
+const ijulia_jsonmime_types = Vector{Union{MIME, AbstractVector{MIME}}}([
     [MIME("application/vnd.vegalite.v2+json"), MIME("application/vnd.vega.v3+json")],
     MIME("application/vnd.dataresource+json"),
 ])
 
-register_mime(x::Union{MIME, Vector{MIME}}) = push!(ijulia_mime_types, x)
-register_jsonmime(x::Union{MIME, Vector{MIME}}) = push!(ijulia_jsonmime_types, x)
+register_mime(x::Union{M, AbstractVector{M}}) where {M <: MIME} = push!(ijulia_mime_types, x)
```

Comment:
This should probably be
```jl
register_mime(x::Union{MIME, Vector{MIME}}) = push!(ijulia_mime_types, x)
register_mime(x::AbstractVector{<:MIME}) = push!(ijulia_mime_types, Vector{MIME}(x))
```
in order to handle a wide range of argument types while keeping `ijulia_mime_types` a narrower type.